### PR TITLE
Error Checking During Build Tests

### PR DESCRIPTION
**Pull request by justinwray**
* Execute hh_client during build tests.

* Currently the PHP built-in getimagesizefromstring function is not in the HHVM upstream hhi files, and therefore generates an error.  In the future, once getimagesizefromstring is added upstream, we should use the hh_client exit status.

### CODE REVIEW

**Comment by justinwray** [comment]
#450 will need to be merged before this PR, or the build tests will fail.

**Comment by justinwray** [comment]
This resolves the outstanding issue from #435.

**Comment by justinwray** [comment]
Note, the build is failing due to the errors resolved in #450.

File: extra/run_tests.sh [hunk 1]
```diff
@@ -29,3 +29,11 @@ hhvm vendor/phpunit/phpunit/phpunit tests
 echo "[+] Deleting test database"
 mysql -u "$DB_USER" --password="$DB_PWD" -e "DROP DATABASE IF EXISTS $DB;"
 mysql -u "$DB_USER" --password="$DB_PWD" -e "FLUSH PRIVILEGES;"
+
+# In the future, we should use the hh_client exit status.
+# Current there are some PHP built-ins not found in the hhi files upstream in HHVM.
+echo "[+] Verifying HHVM Strict Compliance and Error Checking"
```
Comment (gsingh93) [inline extra/run_tests.sh]: Heh, nice hack :P

Can you list which PHP builtins it was complaining about for reference?

Comment (justinwray) [inline extra/run_tests.sh]: Thanks! :)

Looks like right now the only one we're using is:  getimagesizefromstring().

**Pull request merged**